Documentation/QuickInstall/Composer/Index.rst

```diff
@@ -5,44 +5,85 @@
 .. _install-via-composer:
 
 ==========================
-Install TYPO3 via composer
+Install TYPO3 via Composer
 ==========================
 
 .. attention::
 
-   The recommended way of installing TYPO3 is via Composer, as described on
-   this page.
+   The recommended way of installing TYPO3 is via Composer as described on this
+   page.
 
 
-To create a new TYPO3 project use the TYPO3 Base Distribution::
+Using Composer
+==============
 
-	# Download the Base Distribution, the latest "sprint" release (10)
-	composer create-project typo3/cms-base-distribution YourNewProjectFolder
+*NIX Environment
+----------------
+
+In *NIX shells the Asterisk character ``*`` has a special meaning and has to be
+properly escaped by surrounding double quotes ``"``:: shell
+
+   composer create-project vendor/package YourNewProjectFolder "*"
+   composer create-project "vendor/package:*" YourNewProjectFolder
```

Comment:
I realize this was this way before - but why is quoting of the asterisk even explained on this page - it is not used in any of the commands below. Why even use it here? Why not just have the 2 options: command with ^ and the version and no version. 

The page currently, is a bit unreadable because of the many notes and tips. I would try to keep it simple.

Reply:
I absolutly agree. Perhaps moving the whole composer part to https://docs.typo3.org/m/typo3/guide-installation/master/en-us/Composer/Index.html would be better and just keeping here a short note and a link to this page for beginners would be better. Seems that over the time this guide was extended with infos about composer without questioning it's place. Will make a new proposition here...